zebra-test/src/vectors.rs

```diff
@@ -136,6 +136,43 @@ pub const GENERIC_TESTNET_TX: [u8; 2005] = [
     0xfd, 0x09, 0xa3, 0x6b, 0x05,
 ];
 
+pub struct SighashTest {
+    pub raw_transaction: Vec<u8>,
+    pub script: Vec<u8>,
+    pub input_index: u64,
+    pub hash_type: u64,
+    pub branch_id: u64,
```

Comment:
Do we want these types to be like JSON, or match Zcash?

It might be easier to just do the conversions all at once as we deserialize.